sunpy/net/hek/hek.py

```diff
@@ -22,7 +25,9 @@
 __all__ = ['HEKClient', 'HEKTable', 'HEKRow']
 
 DEFAULT_URL = 'https://www.lmsal.com/hek/her?'
+HEK_FILE_PATH = 'sunpy/net/hek/hek_properties.json'
 
+u.add_enabled_aliases({"Day": u.day, "steradian": u.sr, "Steradian": u.sr, "arcseconds": u.arcsec, "Arcsec": u.arcsec, "Arcseconds": u.arcsec, "Deg": u.deg, "degrees": u.deg, "Degrees": u.deg, "sec": u.s})
```

Comment:
Can we not lower case some of these units to match?

Reply:
Yeah sure I guess we can then remove all the aliases we create for the case mismatches.

Reply:
@nabobalis added this change. please have a look

Reply:
As long as it works, it looks good.